examples/src/bin/tlsserver-mio.rs

```diff
@@ -417,79 +417,65 @@ impl OpenConnection {
-
-`http' mode means the server blindly sends a HTTP response on each
-connection.
-
-`forward' means the server forwards plaintext to a connection made to
-localhost:fport.
-
-`--certs' names the full certificate chain, `--key' provides the
-RSA private key.
-
-Usage:
-  tlsserver-mio --certs CERTFILE --key KEYFILE [--suite SUITE ...] \
-     [--proto PROTO ...] [--protover PROTOVER ...] [options] echo
-  tlsserver-mio --certs CERTFILE --key KEYFILE [--suite SUITE ...] \
-     [--proto PROTO ...] [--protover PROTOVER ...] [options] http
-  tlsserver-mio --certs CERTFILE --key KEYFILE [--suite SUITE ...] \
-     [--proto PROTO ...] [--protover PROTOVER ...] [options] forward <fport>
-  tlsserver-mio (--version | -v)
-  tlsserver-mio (--help | -h)
-
-Options:
-    -p, --port PORT     Listen on PORT [default: 443].
-    --certs CERTFILE    Read server certificates from CERTFILE.
-                        This should contain PEM-format certificates
-                        in the right order (the first certificate should
-                        certify KEYFILE, the last should be a root CA).
-    --key KEYFILE       Read private key from KEYFILE.  This should be a RSA
-                        private key or PKCS8-encoded private key, in PEM format.
-    --ocsp OCSPFILE     Read DER-encoded OCSP response from OCSPFILE and staple
-                        to certificate.  Optional.
-    --auth CERTFILE     Enable client authentication, and accept certificates
-                        signed by those roots provided in CERTFILE.
-    --crl CRLFILE ...   Perform client certificate revocation checking using the DER-encoded
-                        CRLFILE. May be used multiple times.
-    --require-auth      Send a fatal alert if the client does not complete client
-                        authentication.
-    --no-resumption     Disable stateful session resumption.
-    --tickets           Support tickets (stateless resumption).
-    --max-early-data BYTES   Support receiving BYTES many bytes with 0RTT
-    --protover VERSION  Disable default TLS version list, and use
-                        VERSION instead.  May be used multiple times.
-    --suite SUITE       Disable default cipher suite list, and use
-                        SUITE instead.  May be used multiple times.
-    --proto PROTOCOL    Negotiate PROTOCOL using ALPN.
-                        May be used multiple times.
-    --verbose           Emit log output.
-    --version, -v       Show tool version.
-    --help, -h          Show this screen.
-";
-
-#[derive(Debug, Deserialize)]
+/// Runs a TLS server on :PORT.  The default PORT is 443.
+///
+/// `echo' mode means the server echoes received data on each connection.
+///
+/// `http' mode means the server blindly sends a HTTP response on each connection.
+///
+/// `forward' means the server forwards plaintext to a connection made to `localhost:fport`.
+///
+/// `--certs' names the full certificate chain, `--key' provides the private key.
```

Comment:
This looks pre-existing, but I think the opening backticks on a number of these lines are mismatched with closing single quotes:

```suggestion
/// Runs a TLS server on :PORT.  The default PORT is 443.
///
/// `echo` mode means the server echoes received data on each connection.
///
/// `http` mode means the server blindly sends a HTTP response on each connection.
///
/// `forward` means the server forwards plaintext to a connection made to `localhost:fport`.
///
/// `--certs` names the full certificate chain, `--key` provides the private key.
```

Reply:
Ah good, catch. These were originally old school GNU-style smart quotes; now this is a markdown doc `` is more suitable.

Reply:
TIL "GNU-style smart quotes" :laughing: :water_buffalo: